.github/workflows/ci.yml

```
@@ -27,6 +27,7 @@ jobs:
       - name: Install APM dependencies
         run: |
           apm ci  # uses locked module. use `apm install` for non-locked
+          npm run build
```

Why not put this into a separate step?

```suggestion
      - name: Build
        run: npm run build
```